### PR TITLE
Update TS grammar and added snippets

### PR DESCRIPTION
**Pull request by torn4dom4n**
From https://github.com/Microsoft/vscode/commit/0347eda9312bae709e8008d858a86900e82c5ffa .

### CODE REVIEW

**Comment by damieng** [comment]
Thanks for this.

The LICENCE for both is the same (MIT) - It seems to be standard practice to list both copyright holders rather than repeat the license.

**Comment by torn4dom4n** [comment]
I removed MIT license of Microsoft!

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-Copyright (c) Microsoft Corporation
```
Comment (damieng) [inline LICENSE.md]: I think we need this line back in though

Comment (torn4dom4n) [reply LICENSE.md]: You rịght! This package is copied from Microsoft so it needs that.

**Comment by damieng** [comment]
This is actually causing build failures - it seems the snippets files are invalid JSON.  I'm going to pull those two files out and issue a point release to get master and the hotfix branch going again.

**Pull request merged**